File: JavaScript solutions/1551. Minimum Operations to Make Array Equal.js
/**
 * @param {number} n
 * @return {number}
 */
var minOperations = function (n) {
  let sum = 0;
  for (let i = 0; i < n; i++) sum += 2 * i + 1;
  sum /= n;
  let ans = 0;
  for (let i = 1; i <= sum; i += 2) ans += Math.abs(sum - i);
  return ans;
};

File: C++ solutions/1551. Minimum Operations to Make Array Equal.cpp
class Solution {
public:
    int minOperations(int n) {
        long long sum=0;
        for(int i=0;i<n;i++) sum+=(2*i)+1;
        sum/=n;
        long long ans=0;
        for(int i=1;i<=sum;i+=2) ans+=abs(sum-i);
        return ans;
    }
};